### PR TITLE
Updates for Dec 13th granite-3.0-8b-lab-community

### PR DESCRIPTION
**Pull request by jjasghar**
These are the direct steps for the build that happened on Dec 13th.

### CODE REVIEW

File: docs/cmb/build_process.md [hunk 1]
```diff
@@ -30,29 +42,52 @@ ilab taxonomy diff
...
+mv /home/instructlab/.local/share/instructlab/phased /home/instructlab/.local/share/instructlab/phased_$DATE
+```
+
+Create the directories you moved away:
+```bash
+mkdir /home/instructlab/.local/share/instructlab/phased
+mkdir /home/instructlab/.local/share/instructlab/datasets
+```
+
+## Add the `insturctlab_community` mixin
```
Comment (nathan-weinberg) [inline docs/cmb/build_process.md:60]:
```suggestion
## Add the `instructlab_community` mixin
```

File: docs/cmb/build_process.md [hunk 2]
```diff
@@ -78,93 +114,24 @@ ilab model evaluate --benchmark mmlu_branch --model ~/.local/share/checkpoints/h
...
-```
-
-Set up (if needed)
-```bash
-python3.11 -m venv venv
-source venv/bin/activate
-pip install vllm
-./run.sh
-```
+## Publish to Huggingface
```
Comment (nathan-weinberg) [inline docs/cmb/build_process.md:117]: This will be an `ilab` feature soon! https://github.com/instructlab/instructlab/pull/2768

Comment (jjasghar) [reply docs/cmb/build_process.md:117]: Nice! Get me `gguf` conversion too eh? ❤️

File: docs/cmb/build_process.md [hunk 3]
```diff
@@ -30,29 +42,52 @@ ilab taxonomy diff
...
+cd ~/.local/share/instructlab/datasets/
+wget https://huggingface.co/datasets/instructlab/InstructLabCommunity/resolve/main/instructlab_community.jsonl
+cd ~
+```
+
 ## Create the data
 ```bash
 # annouce the start of the SDG
-ilab data generate --pipeline full --gpus 8
+ilab data generate --use-legacy-tmpl --pipeline full --gpus 8
```
Comment (cdoern) [inline docs/cmb/build_process.md:71]:
```suggestion
ilab data generate --pipeline full --gpus 8
```

File: docs/cmb/build_process.md [hunk 4]
```diff
@@ -30,29 +42,52 @@ ilab taxonomy diff
...
 # annouce the completion of the SDG
 ```
 
 ## Run the training after the generate is complete
 ```bash
 # annouce the start of the training
-ilab model train --strategy lab-multiphase --phased-phase1-data ~/.local/share/instructlab/datasets/knowledge_train_msgs_XXXXXXX.jsonl --phased-phase2-data ~/.local/share/instructlab/datasets/skills_train_msgs_XXXXXXX.jsonl --skip-user-confirm --pipeline accelerated --force-clear-phased-cache
+ilab model train --use-legacy-tmpl --strategy lab-multiphase --phased-phase1-data /home/instructlab/.local/share/instructlab/datasets/knowledge_train_msgs_*.jsonl --phased-phase2-data /home/instructlab/.local/share/instructlab/datasets/skills_train_msgs_*.jsonl --skip-user-confirm --force-clear-phased-cache
```
Comment (cdoern) [inline docs/cmb/build_process.md:78]:
```suggestion
ilab model train --strategy lab-multiphase --phased-phase1-data /home/instructlab/.local/share/instructlab/datasets/knowledge_train_msgs_*.jsonl --phased-phase2-data /home/instructlab/.local/share/instructlab/datasets/skills_train_msgs_*.jsonl --skip-user-confirm --force-clear-phased-cache
```

File: docs/cmb/build_process.md [hunk 5]
```diff
@@ -30,29 +42,52 @@ ilab taxonomy diff
...
+```
+
+## Add the `instructlab_community` mixin
+For the community build, off the `base` model, you should add the community data set, these are the steps:
+```bash
+cd ~/.local/share/instructlab/datasets/
+wget https://huggingface.co/datasets/instructlab/InstructLabCommunity/resolve/main/instructlab_community.jsonl
+cd ~
+```
+
```
Comment (cdoern) [inline docs/cmb/build_process.md:67]:
```suggestion
## Modify your config
`ilab config edit`

find the general section of your config and ensure it matches the following:

general:
  # Debug level for logging.
  # Default: 0
  debug_level: 0
  # Log format. https://docs.python.org/3/library/logging.html#logrecord-attributes
  # Default: %(levelname)s %(asctime)s %(name)s:%(lineno)d: %(message)s
  log_format: '%(levelname)s %(asctime)s %(name)s:%(lineno)d: %(message)s'
  # Log level for logging.
  # Default: INFO
  log_level: INFO
  # Use legacy IBM Granite chat template (default uses 3.0 Instruct template)
  # Default: False
  use_legacy_tmpl: true 
  
use_legacy_tmpl must be true in order to generate data for and train the granite-3.0-8b-base model
```

**Review by cdoern — COMMENTED** [review]
these are the necessary edits I think. I tried to put ```yaml  around the config block but the templating was messed up

**Pull request merged**